tests/testthat/examples/Test_2023_3/output/taxa.csv

```diff
@@ -1,186 +1,186 @@
-Exocarpos sparteus,,,,,,,,,
-Ficus henneana,,,,,,,,,
-Freesia hybrid,,,,,,,,,
-Freesia laxa,,,,,,,,,
-Geijera salicifolia,,,,,,,,,
-Gentiana wingecarribiensis,,,,,,,,,
-Geranium obtusisepalum,,,,,,,,,
-Geranium purpureum subsp. purpureum,,,,,,,,,
-Gompholobium grandiflorum,,,,,,,,,
-Goodenia varia,,,,,,,,,
-Gossia bidwillii,,,,,,,,,
-Grevillea scortechinii,,,,,,,,,
-Hakea macraeana,,,,,,,,,
-Hemisteptia lyrata,,,,,,,,,
-Hibbertia crinita,,,,,,,,,
-Hibiscus brachysiphonius,,,,,,,,,
-Hibiscus sturtii,,,,,,,,,
-Hypericum elodes,,,,,,,,,
-Hypoxis glabella var. glabella,,,,,,,,,
-Imperata cylindrica,,,,,,,,,
-Indigofera triflora,,,,,,,,,
-Ipomoea cairica,,,,,,,,,
-Isolepis montivaga,,,,,,,,,
-Isotoma axillaris,,,,,,,,,
-Isotoma fluviatilis subsp. fluviatilis,,,,,,,,,
-Juncus acutus subsp. acutus,,,,,,,,,
-Juncus kraussii,,,,,,,,,
-Lantana montevidensis,,,,,,,,,
-Lepidium africanum,,,,,,,,,
-Lepidium foliosum,,,,,,,,,
-Lepidium oxytrichum,,,,,,,,,
-Leptomeria drupacea,,,,,,,,,
-Leptospermum continentale,,,,,,,,,
-Leptospermum obovatum,,,,,,,,,
-Leucopogon amplexicaulis,,,,,,,,,
-Lobelia pedunculata,,,,,,,,,
-Lobelia trigonocaulis,,,,,,,,,
-Lotononis bainesii,,,,,,,,,
-Lotus corniculatus,,,,,,,,,
-Lupinus polyphyllus,,,,,,,,,
-Medicago orbicularis,,,,,,,,,
-Melichrus procumbens,,,,,,,,,
-Mischocarpus australis,,,,,,,,,
-Muehlenbeckia adpressa,,,,,,,,,
-Myoporum boninense subsp. australe,,,,,,,,,
-Nicotiana glauca,,,,,,,,,
-Nicotiana goodspeedii,,,,,,,,,
-Oxalis corniculata,,,,,,,,,
-Pandorea pandorana subsp. austrocaledonica,,,,,,,,,
-Paraserianthes lophantha,,,,,,,,,
-Parentucellia latifolia,,,,,,,,,
-Phlegmatospermum cochlearinum,,,,,,,,,
-Phyla nodiflora,,,,,,,,,
-Picris barbarorum,,,,,,,,,
-Pimelea ligustrina subsp. ligustrina,,,,,,,,,
-Pinus pinea,,,,,,,,,
-Pisum sativum var. arvense,,,,,,,,,
-Pittosporum erioloma,,,,,,,,,
-Plantago turrifera,,,,,,,,,
-Polycarpon tetraphyllum,,,,,,,,,
-Potamogeton australiensis,,,,,,,,,
-Psoralea pinnata,,,,,,,,,
-Pultenaea ferruginea,,,,,,,,,
-Rhodamnia argentea,,,,,,,,,
-Rorippa gigantea,,,,,,,,,
-Rubus parvifolius,,,,,,,,,
-Salix alba,,,,,,,,,
-Schenkia australis,,,,,,,,,
-Schoenus apogon,,,,,,,,,
-Schoenus brevifolius,,,,,,,,,
-Schoenus lepidosperma,,,,,,,,,
-Schoenus vaginatus,,,,,,,,,
-Scleria rugosa,,,,,,,,,
-Senecio longipilus,,,,,,,,,
-Sida goniocarpa,,,,,,,,,
-Sigesbeckia orientalis,,,,,,,,,
-Sisymbrium irio,,,,,,,,,
-Sparaxis bulbifera,,,,,,,,,
-Spergularia diandra,,,,,,,,,
-Sphaeromorphaea littoralis,,,,,,,,,
-Stellaria angustifolia subsp. angustifolia,,,,,,,,,
-Styphelia viridis,,,,,,,,,
-Swainsona cadellii,,,,,,,,,
-Teucrium puberulum,,,,,,,,,
-Thelymitra kangaloonica,,,,,,,,,
-Thesium australe,,,,,,,,,
-Trifolium dubium,,,,,,,,,
-Veronica catenata,,,,,,,,,
-Veronica derwentiana,,,,,,,,,
-Wahlenbergia luteola,,,,,,,,,
+aligned_name,taxon_name,taxonomic_dataset,taxon_rank,taxonomic_status,genus,binomial,trinomial,family,taxon_id,taxon_distribution
+,Abildgaardia ovata,,,,,,,,,
```

Comment:
did you mean to add a blank cell at the start of each row?

Reply:
It was intentional, but open to discussion. I added `aligned_name` to the taxa table, but went back and forth several times as to whether this was a good/bad idea. Now leaning back to removing it.